File: solutions/c++/airlinemanagementsystem/Passenger.cpp
#include "Passenger.hpp"
#include <iostream>

Passenger::Passenger(std::string name, std::string passportNumber, std::string contactNumber)
    : name(name), passportNumber(passportNumber), contactNumber(contactNumber) {}

std::string Passenger::getName() const { return name; }
std::string Passenger::getPassportNumber() const { return passportNumber; }
std::string Passenger::getContactNumber() const { return contactNumber; }

void Passenger::displayInfo() const {
    std::cout << "Passenger Details:" << std::endl;
    std::cout << "Name: " << name << std::endl;
    std::cout << "Passport Number: " << passportNumber << std::endl;
    std::cout << "Contact Number: " << contactNumber << std::endl;
} 

File: solutions/python/ridesharingservice/passenger.py
class Passenger:
    def __init__(self, id, name, contact, location):
        self.id = id
        self.name = name
        self.contact = contact
        self.location = location

    def get_id(self):
        return self.id

    def get_name(self):
        return self.name

    def get_contact(self):
        return self.contact

    def get_location(self):
        return self.location